gulpfile.js

```diff
@@ -1,26 +1,24 @@
-'use strict';
```

Comment:
Ok, good to know. If I get the time, it sounds like you would accept a PR like that then 😄 I can be sure to replicate the behaviour you currently have!